#request
geometry_msgs/PoseStamped   tempPose
---
#response
bool    handover_bool
